test/test_torch.py

```diff
@@ -2683,6 +2683,11 @@ def test_qtensor(self):
         # This calls _th_fill_
         # qr[0] = 8 # float asignment
         # self.assertEqual(qr.item(), 8)
+        # we can also print a qtensor
+        print(qr)
```

Comment:
can you just assert what you expect the print to look like?

Reply:
sure